### DISCUSSION TITLE
Does frigate add timestamp to video recorded?

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#1458] [General]
I turned off timestamp from my cameras to avoid motion/AI detection by frigate.
I set timestamp to true like this in the config for cameras which stamps the time fine.

```
    snapshots:
      enabled: True
      timestamp: True
```

wondering if there is such a setting or possible to add timestamp to clips/recordings, may be a dumb question becaz i could add the timestamp back from the camera's original settings and add a mask in frigate to eliminate motion.

**Comment by blakeblackshear** [comment]
It's not possible because it would require re-encoding the video.

**Reply by balajeek** [reply]
yeah, i thought so. It makes sense, i will enable timestamp from camera settings and mask in frigate.
Thanks anyway.

**Reply by Lensters** [reply]
Maybe the configuration timestamp_style should be moved by default to a sub item of timestamp under snapshots instead of a global item to help avoid confusion?

Also, since I'm masking out all the camera timestamps with a global motion mask, would it be possible to show those in the camera debug view of the gui as well as the camera specific masks?
